sdk/cosmos/azure-spring-data-cosmos/src/test/java/com/azure/spring/data/cosmos/repository/integration/ReactiveLongIdDomainPartitionPartitionRepositoryIT.java

```diff
@@ -78,19 +74,26 @@ public static void afterClassCleanup() {
 
     @Test
     public void testLongIdDomainPartition() {
-        this.repository.deleteAll().block(DEFAULT_TIME_OUT);
-        Assert.assertFalse(this.repository.findById(ID_1).blockOptional(DEFAULT_TIME_OUT).isPresent());
+        Mono<Void> deletedMono = this.repository.deleteAll();
+        StepVerifier.create(deletedMono).thenAwait().verifyComplete();
 
-        this.repository.save(DOMAIN_1).block(DEFAULT_TIME_OUT);
-        Optional<LongIdDomainPartition> foundOptional = this.repository.findById(ID_1).blockOptional(DEFAULT_TIME_OUT);
+        Mono<LongIdDomainPartition> idMono = this.repository.findById(ID_1,
+            new PartitionKey(entityInformation.getPartitionKeyFieldValue(DOMAIN_1)));
+        StepVerifier.create(idMono).verifyComplete();
```

Comment:
Here, we can make sure that nothing is returned from the backend - since we deleted all entities. 

```suggestion
        StepVerifier.create(idMono).expectNextCount(0).verifyComplete();
```